backend/src/apiserver/model/cron_schedule.go

```diff
@@ -0,0 +1,21 @@
+package model
+
+type CronSchedule struct {
+	// Time at which scheduling starts.
+	// If no start time is specified, the StartTime is the creation time of the schedule.
+	CronScheduleStartTimeInSec *int64 `gorm:"column:CronScheduleStartTimeInSec;"`
+
+	// Time at which scheduling ends.
+	// If no end time is specified, the EndTime is the end of time.
+	CronScheduleEndTimeInSec *int64 `gorm:"column:CronScheduleEndTimeInSec;"`
+
+	// Cron string describing when a workflow should be created within the
+	// time interval defined by StartTime and EndTime.
+	Cron *string `gorm:"column:Schedule;"`
+}
+
+func (c CronSchedule) IsEmpty() bool {
+	return c.CronScheduleStartTimeInSec == nil &&
+		c.CronScheduleEndTimeInSec == nil &&
+		(c.Cron == nil || *c.Cron == "")
```

Comment:
Why is the empty case checked for the `Cron` field but not the other fields (e.g. `0` for `CronScheduleStartTimeInSec`)?

Reply:
Good catch!
Added for all fields.